rebar.config

```diff
@@ -2,7 +2,7 @@
 {deps,
  [
   {sext, "1.4.1"},
-  {rocksdb,"0.14.0"}
+  {rocksdb,"0.19.0"}
```

Comment:
you should use the 0.20.1 also

Reply:
update to 0.20.1

Reply:
the aternity team update it to 0.24.0, it should be resolved.